examples/vue-pinia/pages/todos/todo.page.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+  <span>{{ todo?.text }}</span>
+  <br />
+  <a href="/">Back</a>
+</template>
+
+<script setup>
+import { onServerPrefetch, computed } from 'vue'
+import { useTodos } from '../../stores/useTodos'
+
+const props = defineProps(['todoId'])
+
+onServerPrefetch(async () => {
+  const todosStore = useTodos()
+  if (todosStore.todoList.length === 0) {
```

Comment:
Why is that needed?

Reply:
You're right, it's not needed. I'll remove it.